package.json

```diff
@@ -46,9 +46,13 @@
 		"registry": "https://registry.npmjs.org/"
 	},
 	"dependencies": {
+		"case-anything": "^2.1.13",
```

Comment:
is there a way to avoid this dependnecy? if it does not add much kbs then its fine

Reply:
I installed this dependency for their `TrainCase` function, as `radash` does not provide such function.  In the generated schema, some relations like `User-Accounts` is TrainCase. I need to use this function to transform table name when I return the response. I have used named import to import this function, so it should not add bloat to the library.